src/reactapp/src/api/sendRequest.js

```diff
@@ -1,9 +1,17 @@
 import { config } from '../config';
+import LS from '../utils/LS';
```

Comment:
I'm a fan or readable variables and am wondering what advantage we have from shortening LocalStorage :)
Webpack will transpile this to some random variable anyway, so it's not saving bits and bytes in the final application :D

Reply:
I've spent a bit of time naming this stuff. I really weighed `localStorage` and `LS` in my mind and finally opted `LS`.  Don't ask me why I opted it. I just felt in that way. invoking like `LS.saveCustomerToken(customerToken)` rather than `localStorage.saveCustomerToken(customerToken)` will fit in one line and compact i felt. 

If you feel `LS` not giving much sense then i will happily rename it. :)